extensions/wikia/Wall/WallRelatedPages.class.php

```diff
@@ -142,6 +142,14 @@ function getMessagesRelatedArticleTitles( $messageIds, $orderBy = 'order_index'
 	}
 
 
+	/**
+	 * TODO: Used by RelatedForumDiscussionController only, move there?
+	 *
+	 * @param $pageId
+	 * @param $messageCount
+	 * @param $replyCount
+	 * @return array
+	 */
 	public function getArticlesRelatedMessgesSnippet( $pageId, $messageCount, $replyCount ) {
```

Comment:
we can get rid of the $replyCount param as it is not used. What about the TODO above? if it is the only usage I would move it there